api/src/services/application-csv-export.service.ts

```diff
@@ -45,12 +48,15 @@ export const typeMap = {
 export class ApplicationCsvExporterService
   implements CsvExporterServiceInterface
 {
+  readonly dateFormat: string = 'MM-DD-YYYY hh:mm:ssA z';
+  timeZone = process.env.TIME_ZONE;
```

Comment:
nit: since this is an env variable now, unsure if we need it as an instance variable as well